internal/clients/idler.go

```diff
@@ -20,10 +21,18 @@ const (
 	OpenShiftAPIParam = "openshift_api_url"
 )
 
-// status represent status of the service true if idle, false if unidle.
-type status struct {
-	IsIdle bool `json:"is_idle"`
-}
+type podState string
+
+const (
+	// UnknownState used when the state isn't known, along with err
+	UnknownState podState = ""
+	// Running represents a running pod
+	Running = "running"
```

Comment:
Why is `running` not a podState? Can I use podState and string interchangeably?

Reply:
Running is a `podState` because the const started with declaring the first const `UnknownState` as a `podState`